Core/Domain/Recovery/RecoveryState.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Lykke.Service.ClientAccountRecovery.Client.AutoRestClient.Models;
+
+namespace Core.Domain.Recovery
+{
+    public class RecoveryState
```

Comment:
Add a description for the classes in public API

Reply:
Done.